btree/btree.c

```diff
@@ -4692,12 +4970,33 @@ static uint32_t vkvv_inode_rec_key_size(const struct nd *node, int idx)
 static uint32_t vkvv_rec_key_size(const struct nd *node, int idx)
 {
 	struct vkvv_head *h = vkvv_data(node);
-	if (h->vkvv_level == 0)
+	if (vkvv_addrtype_get(node) == INDIRECT_ADDRESSING)
+		return vkvv_indirect_rec_key_size(node, idx);
+	else if (h->vkvv_level == 0)
 		return vkvv_lnode_rec_key_size(node, idx);
 	else
 		return vkvv_inode_rec_key_size(node, idx);
 }
 
+static uint32_t vkvv_indirect_rec_val_size(const struct nd *node, int idx)
+{
+	struct vkvv_head *h    = vkvv_data(node);
+	int               size = h->vkvv_nsize;
+	int               vsize;
+	int               offset;
+
+	if (h->vkvv_level == 0) {
+		void **p_v_addr;
+		void  *v_addr;
+		vsize  = INDIRECT_VAL_SIZE;
```

Comment:
In the below assignments,  we see random alignment of =.
In a block of code, either align all the = or don't align any of them.
Upto the developers choice.
Take similar care for other places.

Reply:
new changes are added in this function.